### PR TITLE
[YUNIKORN-2166] Translate 1.4 release notes and roadmap to zh-cn

### PR DESCRIPTION
**Pull request by Haser0305**
### What is this PR for?
Translate 1.4 release notes and roadmap to zh-cn

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [x] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2166

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-pages/release-announce/1.4.0.md [hunk 1]
```diff
@@ -33,77 +33,65 @@ Apache YuniKorn社区在此版本中修复了270个[JIRAs](https://issues.apache
...
+追踪和故障排除应用程式是专门针对 YuniKorn。 Kubernetes 中没有应用程式的概念。新增了事件系统来公开应用程式、队列和节点的变更以公外部使用。
 
-Please see [YUNIKORN-1628](https://issues.apache.org/jira/browse/YUNIKORN-1628)
+详细资讯请参阅 [YUNIKORN-1628](https://issues.apache.org/jira/browse/YUNIKORN-1628)
 
-### Build updates
-The YuniKorn builds no longer relies on centrally installed tools while building the binaries or images.
-Each build will download and maintain its own set of tools, not even shared between checked out repositories.
+### 建置更新
+在建置二进位档案或映像时，YuniKorn 建置不再依赖集中安装的工具。每个版本都会下载并维护自己的工具集，甚至不会在签出的除从库之间共用。
```
Comment (FrankYang0529) [inline i18n/zh-cn/docusaurus-plugin-content-pages/release-announce/1.4.0.md:87]:
```suggestion
在建置二进位档案或映像时，YuniKorn 建置不再依赖集中安装的工具。每个版本都会下载并维护自己的工具集，甚至不会在签出的储存库之间共用。
```

**Review by FrankYang0529 — CHANGES_REQUESTED** [review]

File: i18n/zh-cn/docusaurus-plugin-content-pages/release-announce/1.4.0.md [hunk 2]
```diff
@@ -33,77 +33,65 @@ Apache YuniKorn社区在此版本中修复了270个[JIRAs](https://issues.apache
...
-The change was announced on the dev@ mailing list in late July.
+这项变更于 7 月底在 dev@ 邮件单上宣布。
 
-Please see these jiras for details:
+请参阅这些 jiras 已了解详细资讯:
 - k8shim: [YUNIKORN-1862](https://issues.apache.org/jira/browse/YUNIKORN-1862) and [YUNIKORN-1865](https://issues.apache.org/jira/browse/YUNIKORN-1865)
 - core: [YUNIKORN-1868](https://issues.apache.org/jira/browse/YUNIKORN-1868)
 - web UI: [YUNIKORN-1873](https://issues.apache.org/jira/browse/YUNIKORN-1873)
 
 ## Community
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-pages/release-announce/1.4.0.md:96]: 社区

**Review by 0yukali0 — CHANGES_REQUESTED** [review]

File: i18n/zh-cn/docusaurus-plugin-content-pages/release-announce/1.4.0.md (hunk 1 above)
Comment (Haser0305) [reply i18n/zh-cn/docusaurus-plugin-content-pages/release-announce/1.4.0.md]: Thank you for your correction, i have corrected it.

File: i18n/zh-cn/docusaurus-plugin-content-pages/release-announce/1.4.0.md (hunk 2 above)
Comment (Haser0305) [reply i18n/zh-cn/docusaurus-plugin-content-pages/release-announce/1.4.0.md]: Thank you for reminding, i have corrected it.

**Review by FrankYang0529 — APPROVED** [review]
+1. Thank you.

**Review by hUkfeq-sihwys-9pezqy — CHANGES_REQUESTED** [review]
1.24

**Comment by wilfred-s** [comment]
committing based on @FrankYang0529 approval

**Pull request closed without merging**